src/helpers/constants.h

```diff
@@ -6,19 +6,13 @@
 #define GODOTFMOD_CONSTANTS_H
 
 #include <fmod_common.h>
+#include <godot_cpp/core/class_db.hpp>
 
 #define DECLARE_CONSTANT(name)\
-    int _##name = name;\
-    int get_##name(){\
-        return _##name;\
-    }\
-    void set_##name(int i){ \
-        _##name = name;\
-        return;\
-    }\
+    GD_##name = name,\
```

Comment:
Instead of defining getters and setters like for GDNative, I am now simply defining the enum constant like so (I have to use some sort of prefix, as macros complain otherwise...)